sympy/combinatorics/fp_groups.py

```diff
@@ -933,7 +933,7 @@ def simplify_presentation(*args, **kwargs):
                                               change_gens=change_gens)
         if gens:
             return FpGroup(gens[0].group, rels)
-        return FpGroup([])
+        return FpGroup(free_group([]), [])
```

Comment:
```
>>> from sympy.combinatorics.free_groups import free_group
>>> from sympy.combinatorics.fp_groups import FpGroup
>>> FpGroup(free_group([]), [])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "sympy/combinatorics/fp_groups.py", line 61, in __init__
    self.generators = self._generators()
  File "sympy/combinatorics/fp_groups.py", line 78, in _generators
    return self.free_group.generators
AttributeError: 'tuple' object has no attribute 'generators'
```

`free_group([])` returns a tuple (of length 1). `FreeGroup([])` should be used instead.